### ISSUE TITLE
Font size issues

### ISSUE THREAD

**Issue by EtienneBruines** [EngoEngine/engo#381]
- [x] Font atlas is generated/cached only once per font (and not per font-size; so if you need a higher resolution, you're out of options);
- [x] The `SpaceComponent` is completely ignored w.r.t. the size / area in which to draw the text. `font.Size` is the only thing that currently defines the size of the text.

**Comment by Noofbiz** [comment]
#488 is a duplicate of part two of this

**Comment by Noofbiz** [comment]
Fixed in #499
